official/bert/benchmark/bert_benchmark.py

```diff
@@ -70,7 +71,10 @@ def _run_bert_classifier(self, callbacks=None):
     eval_steps = int(
         math.ceil(input_meta_data['eval_data_size'] / FLAGS.eval_batch_size))
     strategy = distribution_utils.get_distribution_strategy(
-        distribution_strategy='mirrored', num_gpus=self.num_gpus)
+        distribution_strategy='mirrored' if use_ds else 'noop',
+        num_gpus=self.num_gpus)
```

Comment:
It would be great to follow the pattern of the other models and use "off" which then triggers dist strat to be None like here:
https://github.com/tensorflow/models/blob/master/official/utils/misc/distribution_utils.py#L121

The test I want to duplicate is "off" to match the other tests.

It does mean your with scope (if used) needs to also call the util to get a scope that is not None or some other trick.  If Priya wants "off" to be tf.distribute.get_strategy() then we just updated the code.